cmd/status.go

```diff
@@ -19,34 +19,40 @@ package cmd
 
 import (
 	"fmt"
-	"net/url"
-
-	"github.com/andricdu/go-song/song"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+var pFlag bool
+
 func init() {
 	RootCmd.AddCommand(statusCmd)
+	statusCmd.Flags().BoolVarP(&pFlag, "ping", "p", false, "Just check if server is alive")
```

Comment:
Did you mean "checking if server ..."?

Reply:
No, I meant "[This command will] just check if [the] server is alive". :-)

Reply:
:+1: